lib/ts/recipe/dashboard/constants.ts

```diff
@@ -34,6 +34,9 @@ export const USERROLES_ROLE_API = "/api/userroles/role";
 export const USERROLES_PERMISSIONS_API = "/api/userroles/role/permissions";
 export const USERROLES_REMOVE_PERMISSIONS_API = "/api/userroles/role/permissions/remove";
 
+export const CREATE_EMAIL_PASSWORD_USER = "/api/user/create/emailpassword";
+export const CREATE_PASSWORDLESS_USER = "/api/user/create/passwordless";
```

Comment:
```suggestion
export const CREATE_EMAIL_PASSWORD_USER = "/api/user/emailpassword";
export const CREATE_PASSWORDLESS_USER = "/api/user/passwordless";
```

Reply:
done

Reply:
TODO: need to change the folder structure as well. But no doing right now as moving files from a folder will create a new files and it would not be easy to review the changes.